src/event.rs

```
@@ -511,19 +511,12 @@ where
 				) {
 					Ok(final_tx) => {
 						let needs_manual_broadcast =
-							match self.liquidity_source.as_ref().map(|ls| {
+							self.liquidity_source.as_ref().map_or(false, |ls| {
```

Don't you want to surface the error in the log?